app/helpers/layout_helper.rb

```diff
@@ -95,8 +95,13 @@ def stylesheet(*args)
     content_for(:stylesheets) { stylesheet_link_tag(*args) }
   end
 
-  def javascript(*args)
-    content_for(:javascripts) { javascript_include_tag(*args) }
+  def javascript(*args, overrides)
```

Comment:
Is this even syntactically valid? Should it be
```suggestion
  def javascript(*args, **overrides)
```

Reply:
syntactically valid? Dont know
**overrides didnt work when I tried it

Reply:
Apparently it is, but I was surprised when I saw it

Reply:
Well, Ruby is indeed as a natural language, since everything you write needs a context :melting_face:

Even though it's not conventional (please, don't put splat/rest arguments (*args) before required ones) it works, but... backwards. The last passed argument to `javascript` will be extracted into `overrides`.

But be aware that by adding required parameter, `javascript` method can be no longer used without any arguments.

Reply:
changed to **kwargs